db.go

```diff
@@ -330,20 +338,28 @@ func (x *dummyUserStore) GetIdentities(getIdentitiesFlag GetIdentitiesFlag) ([]A
 		if (getIdentitiesFlag&GetIdentitiesFlagDeleted == 0) && v.archived {
 			continue
 		}
-		list = append(list, AuthUser{v.userId, v.email, v.username, v.firstname, v.lastname, v.mobilenumber, v.telephonenumber, v.remarks, v.created, v.createdby, v.modified, v.modifiedby, v.authUserType, v.archived, v.passwordModifiedDate})
+		list = append(list, AuthUser{v.userId, v.email, v.username, v.firstname, v.lastname, v.mobilenumber, v.telephonenumber, v.remarks, v.created, v.createdby, v.modified, v.modifiedby, v.authUserType, v.archived, v.passwordModifiedDate, v.accountLocked})
 	}
 	return list, nil
 }
 
+func (x *dummyUserStore) LockAccount(userId UserId) error {
+	return nil
```

Comment:
I don't know what the value would be... but this should probably be:
```go
user := x.users[userId]
if user.archived {
    return // some Error
}

user->accountLocked = false
return nil
```

Reply:
That looks right. And if you make these suggested changes, then the unit test will probably work regardless of whether it's postgres_backend or not